### DISCUSSION TITLE
View metrics admin access issue | XMLA endpoint connection to Datamart's semantic model

### DISCUSSION THREAD

**Discussion by Karthikkumar89** [DaxStudio/DaxStudio#1246] [Q&A] [answered]
Team, I have Member access to a Power BI premium workspace contains a Datamart & its Semantic model.
I am able to connect from DAX STUDIO (3.0.11) to model using XMLA end point but when I hit VIEW METRICS it throws error saying
"You do not have admin rights to the current data model. Admin rights are required to view the metrics"

**Comment by dgosbell** [comment, marked as answer]
I answer this in the duplicate of this question that you posted as issue #1245

**Reply by Karthikkumar89** [reply]
Thank you
